config/default/block.block.my_awesome_theme_weatherdata.yml

```diff
@@ -22,4 +22,5 @@ settings:
   level: 1
   depth: 0
   expand_all_items: false
+  key: 8795cf1702a915bf4f6e1c1ca54fed35
```

Comment:
If I remember correctly this is a menu block, and it is disabled.
Why this key is here then?
Why you even need this block?